lib/routes/account.js

```diff
@@ -1021,50 +1045,78 @@ module.exports = function (
         validate: {
           payload: {
             uid: isA.string().max(32).regex(HEX_STRING).required(),
-            code: isA.string().min(32).max(32).regex(HEX_STRING).required()
+            code: isA.string().min(32).max(64).regex(HEX_STRING).required()
```

Comment:
> Also, deciding which path to take based an implicit detail like token-length seems fragile to me.

This was my thought too. Can the tokens be one and the same? When verifying an account, it should verify the session (though not all sessions), and when verifying a session, it should verify an account.